pwa/src/service-worker/service-worker.js

```diff
@@ -37,11 +37,8 @@ export function setUpServiceWorker() {
     // TODO: control with env var
     self.__WB_DISABLE_DEV_LOGS = true
 
-    // Globals
+    // Globals (Note: global state resets each time SW goes idle)
 
-    // Will be populated upon activation with a promise that accesses the
-    // recorded sections IndexedDB using the `idb` library - see `createDB()`
-    self.dbPromise
     // Tracks recording states for multiple clients to handle multiple windows
     // recording simultaneously
     self.clientRecordingStates = {}
```

Comment:
Should we store `clientRecordingStates` in session/local storage or is this state short lived?

Reply:
I left this as-is because it will only be populated with content for the duration of a recording, which is pretty brief, so it's fine to reset it to `{}` on termination/startup, I believe.

That said, in the testing session last week, [Hendrik encountered an issue](https://dhis2.slack.com/archives/CJF3MDK8Q/p1629373941003500) that seemed to be related to `clientRecordingStates`. I haven't been able to replicate it, and it's possible those error messages were logged in the process of other things failing